### PR TITLE
OGRCT: Export novel OGRCoordinateTransformation functions to C API

### PR DESCRIPTION
**Pull request by drons**

### CODE REVIEW

File: gdal/ogr/ogrct.cpp [hunk 1]
```diff
@@ -960,6 +960,66 @@ OCTClone(OGRCoordinateTransformationH hTransform)
...
 }
 
+/************************************************************************/
+/*                             OCTGetSourceCS()                         */
+/************************************************************************/
+
+/**
+ * Transformation's source coordinate system reference.
+ *
+ * This is the same as the C++ function OGRCreateCoordinateTransformation::GetSourceCS
```
Comment (rouault) [inline gdal/ogr/ogrct.cpp:970]: It should be mentioned that the returned object might be null, and is owned by hTransform and should not be freed.

File: gdal/ogr/ogrct.cpp [hunk 2]
```diff
@@ -960,6 +960,66 @@ OCTClone(OGRCoordinateTransformationH hTransform)
...
+}
+
+/************************************************************************/
+/*                             OCTGetTargetCS()                         */
+/************************************************************************/
+
+/**
+ * Transformation's target coordinate system reference.
+ *
+ * This is the same as the C++ function OGRCreateCoordinateTransformation::GetTargetCS
```
Comment (rouault) [inline gdal/ogr/ogrct.cpp:990]: It should be mentioned that the returned object might be null, and is owned by hTransform and should not be freed.

File: gdal/ogr/ogrct.cpp [hunk 3]
```diff
@@ -960,6 +960,66 @@ OCTClone(OGRCoordinateTransformationH hTransform)
...
+}
+
+/************************************************************************/
+/*                             OCTGetInverse()                          */
+/************************************************************************/
+
+/**
+ * Inverse transformation object.
+ *
+ * This is the same as the C++ function OGRCreateCoordinateTransformation::GetInverse
```
Comment (rouault) [inline gdal/ogr/ogrct.cpp:1010]: It should be mentioned that the returned object might be null, and should be freed with OCTDestroyCoordinateTransformation()

**Comment by rouault** [comment]
I'd like to see minimal testing of this as well

**Comment by drons** [comment]
Thanks!

**Pull request merged**